src/client/renderingengine.cpp

```diff
@@ -152,6 +152,9 @@ RenderingEngine::RenderingEngine(IEventReceiver *receiver)
 	driver = m_device->getVideoDriver();
 	infostream << "Using the " << RenderingEngine::getVideoDriverInfo(driver->getDriverType()).friendly_name << " video driver" << std::endl;
 
+	// This changes the minimum allowed number of vertices in a VBO. Default is 500.
+	driver->setMinHardwareBufferVertexCount(4);
```

Comment:
is there a minimum limit where VBOs don't make sense (perform worse)? or any downside of creating a lot of them?
if so maybe 50 would be better

Reply:
I am not aware of any downside to having everything as a VBO. The ids are GLuint, so there would be a hard limit of 2^32 buffers. There is probably some allocation overhead, but minetest is not creating millions of buffers, just thousands.

Some evidence to this effect: WebGL and WebGL2 do not support client-side arrays for any draw operations. Buffers are always required for both vertices and indices.